src/TiledArray/tensor/tensor.h

```diff
@@ -282,8 +298,18 @@ class Tensor {
   template <typename T1, typename T2, typename Op,
             typename std::enable_if<is_tensor<T1, T2>::value>::type* = nullptr>
   Tensor(const T1& left, const T2& right, Op&& op, const Permutation& perm)
-      : pimpl_(std::make_shared<Impl>(perm * left.range())) {
+      : pimpl_(std::make_shared<Impl>(perm.outer_permutation() * left.range())) {
```

Comment:
@ryanmrichard I noticed that only some constructors were converted to support ToT functionality? Any reason?

Reply:
I more-or-less found what parts of the code needed changed by adding the ToTs and seeing what broke. My guess is the ToT code I wrote never hit those ctors so I never knew they needed changing.